clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -1286,6 +1282,11 @@ class KernelObjVisitor {
     for (const auto Field : Owner->fields())
       visitField(Owner, Field, Field->getType(), Handlers...);
   }
+
+  template <typename... HandlerTys>
+  void visitArray(const CXXRecordDecl *Owner, FieldDecl *Field,
+                  QualType ArrayTy, HandlerTys &...Handlers);
+
```

Comment:
I assume this code move was inadvertent?

Reply:
It was a private method I moved to be public.